### PR TITLE
Fixed the recipe gcp_logging_ts and processor gcp_logging_timesketch

### PR DESCRIPTION
**Pull request by roshanmaskey**
Fixed the processor `gcp_logging_timesketch` and updated the recipe `gcp_logging_ts` to use the processor - `gcp_logging_ts` did not use any processor.

### CODE REVIEW

File: dftimewolf/lib/processors/gcp_logging_timesketch.py [hunk 1]
```diff
@@ -246,11 +243,12 @@ def _BuildMessageString(self, timesketch_record: Dict[str, Any]) -> None:
 
     timesketch_record['message'] = message
 
-  def _ProcessLogContainer(self, logs_container: containers.GCPLogs) -> None:
+  def _ProcessLogContainer(self, logs_container: containers.File) -> None:
     """Processes a GCP logs container.
 
     Args:
-      logs_container (GCPLogs): logs container.
+      logs_container (containers.File): container containing GCPLogsCollector
```
Comment (ramo-j) [inline dftimewolf/lib/processors/gcp_logging_timesketch.py:250]: Nit: You don't need the type annotation in the comments when the dunction definition is typed.

File: dftimewolf/lib/processors/gcp_logging_timesketch.py [hunk 2]
```diff
@@ -262,23 +260,22 @@ def _ProcessLogContainer(self, logs_container: containers.GCPLogs) -> None:
...
+            line, logs_container.name)
         if transformed_line:
           output_file.write(transformed_line)
           output_file.write('\n')
     output_file.close()
 
-    timeline_name = 'GCP logs {0:s} "{1:s}"'.format(
-        logs_container.project_name, logs_container.filter_expression)
+    current_timestamp = datetime.utcnow().strftime('%Y%m%d%H%M%S')
+    timeline_name = f'{current_timestamp}'
```
Comment (ramo-j) [inline dftimewolf/lib/processors/gcp_logging_timesketch.py:270]: Perhaps timeline name should be a bit more descriptive than just the timestamp?

File: dftimewolf/lib/processors/gcp_logging_timesketch.py (hunk 1 above)
Comment (roshanmaskey) [reply dftimewolf/lib/processors/gcp_logging_timesketch.py:250]: Removed type annotation from comment.

File: dftimewolf/lib/processors/gcp_logging_timesketch.py (hunk 2 above)
Comment (roshanmaskey) [reply dftimewolf/lib/processors/gcp_logging_timesketch.py:270]: Updated to use `project_id` in timeline name i.e. `<project_id>_<yyyymmddhhmmss>`.

**Review by ramo-j — APPROVED** [review]
LGTM

**Review by ramo-j — APPROVED** [review]

**Pull request merged**